afl-showdsf.c

```diff
@@ -569,7 +569,17 @@ static void run_target() {
 
   memset(trace_bits, 0, MAP_SIZE);
   memset(dsf_map, 0, dsf_len_actual * sizeof(u32));
```

Comment:
Similar thing here. Does it make sense to simply do`memset(dsf_map, dsf_initial, ...)`? Does that address the issue you've discovered?